### ISSUE TITLE
File input field align error

### ISSUE THREAD

**Issue by mingkim** [Dogfalo/materialize#1596]
![2015-06-26 10 01 41](https://cloud.githubusercontent.com/assets/8112305/8368874/5dcbdf86-1beb-11e5-92be-4b8344a1c6bf.png)

Like this picture, The file input filed align is not correct.
Recently, some guy wrote about this issue and said this belongs to location of modal definition ..
I also use modal in this template file.
I did not change anything different.. What should I have to do?

**Comment by jots** [comment]
I just ran into this today.  documentation is missing ```<div class="file-path-wrapper">```
try:
```html
<div class="row">
        <div class="file-field input-field col s12">
          <div class="btn">
            <span>File Upload</span>
            <input type="file">
          </div>
          <div class="file-path-wrapper">
            <input class="file-path" type="text">
          </div>
        </div>
      </div>
```

**Comment by Dogfalo** [comment]
Fixed in 162cf4a1820
